lib/default.rb

```diff
@@ -4,6 +4,19 @@
 include Nanoc::Helpers::Rendering
 include Nanoc::Helpers::XMLSitemap
 
+# TODO: Remove this monkey-patch if https://github.com/ruby/ruby/pull/3117 get merged.
```

Comment:
This was necessary because net/http fails if the status line is in the HTTP/2 format. This PR is open and we can remove this monkey-patch when (and if) it gets merged.

Reply:
I suggest you use `--http1.1`. I suspect not all programming languages will be able to read HTTP/2 fixtures.
https://ec.haxx.se/http/http-versions

Reply:
updated!